source/partials/next_release/_release.md.erb

```diff
@@ -1,73 +1,49 @@
 
-* <%= link_to_commit 'f501efc5a00519461fdf7d93081f83fcca7df2b9', 'Move to logback for logging', 3831 %>
-  * As a part of this change, `*-log4j.properties` have been moved over to `*-logback.xml`.
-  * This migration solves a number of logging related issues such as <%= link_to_issue 3588%>, <%= link_to_issue 3557%> and <%= link_to_issue 3403%>
+<h4>Security Fixes</h4>
+* <%= link_to_commit '488712d7262dc6dd082aad3f2cf2d3ed05a3caa3', 'Token based approach for agent registration', 4014%>
 
-* <%= link_to_commit 'a8d09f30be8947165696083039146a33a06fc5a0', 'Provides support for parameters and template references in config repo pipelines.' %>
-* <%= link_to_commit '36b675e24c7bc7185048f46499c699f9a49ff8e4', 'Introduces support for bitbucket webhook' %>
-* <%= link_to_commit '6320d55722cde007995579d4fe6ec786b6287ea6', 'Introduces support for gitlab webhook' %>
-* <%= link_to_commit '2cb4c003353e0b4b421af50680d6001cfa864a1a', 'Makes Go Server session timeout configurable', 3856 %>
-* <%= link_to_commit '4e5eeda6eced3defed84a85761cf0f101945b90c', 'Updates footer links to point to installed version\'s api and docs pages', 3837 %>
-* <%= link_to_commit 'e51f3b6427d2b44588d6a4c6a1d0a832b2b8ad82', 'GoCD Agent supports configuring proxies', 995 %>
-* <%= link_to_commit 'c7ec496fe8967206a28902e0f94aae033d5c2386', 'Links pipeline group names in dashboard to corresponding pipeline group settings page for an admin or pipeline group admin.', 3795 %>
+  * Adds a new endpoint to get token from server in exchange of agent uuid. The token for a given agent is issued for unknown agents only - already registered or pending agents cannot request for a token.
+  * Agent requests for a token before registration if required. This token is stored on agent's disk and reused for registration calls.
 
-<h4>Bug Fixes</h4>
-* <%= link_to_commit 'cef6649637d4de2d7bedb6b00f0fa759cb310fe6', 'Fixes a race condition during Job reschedule', 3891 %>
-* <%= link_to_commit 'a1f99453ca629c18753a8835e406e621fa1cb560', 'Improves checksums for agent jar(s) download', 3884 %>
-* <%= link_to_commit '4db7e52bed39e539e4c6f71e789bee188e3e5bd2', 'Does not show the settings icon for config repo pipelines on VSM.', 3892 %> (<%= link_to_issue 3881%>)
-* <%= link_to_commit '686c76d62ad18ae8ffa1299bef2f82e3a225bb4c', 'Fixes issue with users authenticated using WebBasedAuthentication not assigned the right permissions', 3878 %>
 
 <h4>API Changes</h4>
-* <%= link_to_commit 'cba778c7770295fad4d036c82a7802caf41aa723', 'Introduces health check API for Server.', 3912 %>
-* <%= link_to_commit '97df2bfa2578349193c26250d2be59c0909dce58', 'Introduces health check api for agent', 3781 %> (<%= link_to_issue 3074%>)
-* <%= link_to_commit 'e4d041b2340ee6b7928a5fb11b6d3d787665dd1f', 'Adds v2 of Users API', 3720 %>
+* <%= link_to_commit 'd512703973e3ff1d9e1ea8fb5c3e94592504ec71', 'Fix config repo material representers to show only required fields', 3933%>
```

Comment:
We are introducing config repo API this time, right? Don't say that we fixed the config repo API.  and Link to the api docs. There's a helper method called `link_to_api`